backend/app/models/user.py

```
@@ -1,13 +1,53 @@
+from operator import index
```

Ruff (0.11.9)

`operator.index` imported but unused

Remove unused import: `operator.index`

(F401)